components/brave_rewards/resources/ui/components/checkout/creditCardForm/style.ts

```diff
@@ -0,0 +1,77 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+
+export const Container = styled.div`
+  max-width: 270px;
+  margin: 32px auto 10px;
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+
+  input {
```

Comment:
move this in separate styled component

Reply:
Unfortunately, the version of styled-components that we're on doesn't support `ref` the way that I need. We can probably take care of this when we fix the other `input` issues for this component.

Reply:
ok 👍